### DISCUSSION TITLE
Welcome to LogisticsPipes Discussions!

### DISCUSSION THREAD

**Discussion by bziemons** [RS485/LogisticsPipes#1556] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by EpicCraft1997** [comment]
Cant Believe there's not a single comment here, well anyway I'm just curious about will LogisticsPipes be updated to 1.16 any time soon?

**Reply by bziemons** [reply]
I don't think anyone can answer that question at this point. It is not "in reach" and a couple of steps of updates and changes.
